src/main.js

```diff
@@ -1,4 +1,5 @@
 import '@babel/polyfill';
+import "./libs/utils"; // To import some missing functions
```

Comment:
This is cleary another polyfill, so it would be better to name it more explicitly, like `array-find-polyfill`.

Reply:
Already provided by babel actually so removing the file and the import.